src/System.Private.ServiceModel/tests/Common/src/BridgeClient.cs

```diff
@@ -50,8 +50,10 @@ private static void MakeConfigRequest()
             using (HttpClient httpClient = new HttpClient())
             {
                 httpClient.BaseAddress = new Uri(BridgeBaseAddress);
+                string resourceFolder = TestProperties.GetProperty(TestProperties.BridgeResourceFolder_PropertyName);
+                string contentPayload = "{ resourcesDirectory : \"" + resourceFolder + "\" }"; 
```

Comment:
I use string addition here because String.Format throws FormatException for the squiggly braces.  In a coming PR, this payload will be a set of name value pairs and will replace this concat statementt.